iothub/service/src/Microsoft.Azure.Devices.csproj

```diff
@@ -52,6 +52,7 @@
   <!--Nuget package dependencies-->
   <ItemGroup>
     <PackageReference Include="Azure.Core" Version="1.22.0" />
+    <PackageReference Include="Microsoft.Azure.Core.NewtonsoftJson" Version="1.0.0" />
```

Comment:
We have to take this one, huh?

Reply:
The [class we need](https://github.com/Azure/azure-sdk-for-net/blob/main/sdk/core/Microsoft.Azure.Core.NewtonsoftJson/src/NewtonsoftJsonETagConverter.cs) is rather small but so is the dependency. I could go either way on this one

Reply:
Not exposed to the user, though?

Reply:
Might just copy that code.